cmd/authd/daemon/daemon.go

```diff
@@ -49,7 +49,8 @@ type daemonConfig struct {
 func New() *App {
 	a := App{ready: make(chan struct{})}
 	a.rootCmd = cobra.Command{
-		Use:                                                                                 fmt.Sprintf("%s COMMAND", cmdName),
+		Use: fmt.Sprintf("%s COMMAND", cmdName),
+
```

Comment:
Nitpick: unnecessary blank line

Reply:
Well, it's mostly not to have to split the line

Reply:
Why would you have to split the line? Is `gofmt` getting confused with the `i18n.G` tags?

Reply:
Changed this, removing the `i18n.G` and moved as TODO.